projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-results/search-ai-results.component.ts

```diff
@@ -192,4 +199,22 @@ export class SearchAiResultsComponent extends PageComponent implements OnInit, O
         () => (this._hasAnsweringError = true)
       );
   }
+
+  private aiSearchRetryWhen<T>(errors: Observable<T>): Observable<T> {
```

Comment:
Do we need <T> type? It's used only in 2 places and I guess each of these places has the same type so no need to make that function as generic probably?